cdap-ui/app/cdap/components/EntityListView/EntityListHeader/index.js

```diff
@@ -100,31 +95,28 @@ export default class EntityListHeader extends Component {
           tag='div'
           className="sort-toggle"
         >
-          {
-            this.state.activeSort ?
-              <span>{this.state.activeSort.displayName}</span>
-            :
-              'Relevance'
-          }
+          <span>{this.state.activeSort.displayName}</span>
           <span className="fa fa-angle-down float-xs-right"></span>
         </DropdownToggle>
         <DropdownMenu>
           {
             this.state.sortOptions.map((option, index) => {
-              return (
-                <DropdownItem
-                  key={index}
-                  onClick={this.props.onSortClick.bind(this, option)}
-                >
-                  {option.displayName}
-                  {
-                    this.state.activeSort.fullSort === option.fullSort ?
-                      <span className="fa fa-check float-xs-right"></span>
-                    :
-                      null
-                  }
-                </DropdownItem>
-              );
+              if (index !== 0) {
```

Comment:
If this is true for all the cases then we might as well have it as,

```
this.state.sortOptions.slice(1).map((option) => {...})
```

Simpler?